custom/conf/app.example.ini

```diff
@@ -1531,6 +1531,8 @@ LEVEL = Info
 ;; userid = use the userid / sub attribute
 ;; nickname = use the nickname attribute
 ;; email = use the username part of the email attribute
+;; email-normalized = use the username part of the email attribute with single-quotes  
```

Comment:
I think it doesn't need to introduce a new option.

Since the old `email` option doesn't accept invalid chars, I think it is safe to extend its functionality, just make `email` do the "normalization"?

Reply:
And, not only the `email` option needs `NormalizeUserName`, I think others like `nickname` also needs `NormalizeUserName`.

Reply:
Done